python/tutorials/07-extern-functions.py

```diff
@@ -60,9 +62,14 @@ def asin_kernel(
 #  Customize the libdevice library path
 # -------------------------------------
 # We can also customize the libdevice library path by passing the path to the `libdevice` library to the `asin` kernel.
+# This example uses the path from the `third_party/nvidia/backend/lib` directory (the default path).
+
+libdir = Path(__file__).parent.parent.parent / 'third_party/nvidia/backend/lib'
+extern_libs = {}
+extern_libs['libdevice'] = os.getenv("TRITON_LIBDEVICE_PATH", str(libdir / 'libdevice.10.bc'))
```

Comment:
Why is this necessary? The tutorial runs fine for me without this and we are now making it Nvidia specific

Reply:
The current example does NOT show how to pass a customized libdevice library path from line 62 and below
https://github.com/triton-lang/triton/blob/main/python/tutorials/07-extern-functions.py#L62

Added some modifications to include an example for AMD.